libs/features/mas/src/checkout.js

```diff
@@ -55,15 +55,15 @@ export function Checkout({ settings, providers }) {
             extraOptions,
             ...rest
         } = Object.assign(options, placeholder?.dataset ?? {}, overrides ?? {});  
-        let workflowStep = toEnumeration(checkoutWorkflowStep, CheckoutWorkflowStep, Defaults.checkoutWorkflowStep);
+
         options = omitProperties({
             ...rest,
             extraOptions,
             checkoutClientId,
             checkoutMarketSegment,
             country,
             quantity: toQuantity(quantity, Defaults.quantity),
-            checkoutWorkflowStep: workflowStep,
+            checkoutWorkflowStep: toEnumeration(checkoutWorkflowStep, CheckoutWorkflowStep, Defaults.checkoutWorkflowStep),
```

Comment:
this change is likely cosmetic and could be omitted from this PR to avoid build conflict on the bundles.